libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -57,8 +61,11 @@ void AP_Camera_Backend::update()
         return;
     }
 
-    // check GPS status
-    if (AP::gps().status() < AP_GPS::GPS_OK_FIX_3D) {
+    Location current_loc;
+    const AP_AHRS &ahrs = AP::ahrs();
+    if (!ahrs.get_location(current_loc)) {
+        last_location.lat = 0;
```

Comment:
I love to see this change but I wonder why we need to set last_location here

Reply:
I copied that pattern from a clause above.

Which was a mistake; I've removed it.

GPS is still used in here for a timestamp.  The date should probably come from `AP::rtc()`, but we probably also need to have an option bit or something...